src/modules/EventList/index.tsx

```diff
@@ -226,7 +226,9 @@ const DefaultEventRow = ({
                                 metaData={{
                                     createdBy,
                                     eventDate: eventDate ? eventDate.toISO() : undefined,
+                                    eventStream: event.audioStreamUri,
                                     eventType: event.eventType,
+                                    isLive: event?.isLive ? true : false,
```

Comment:
No need to change if you prefer the ternary, but this could also be `!!event?.isLive`